astropy/coordinates/representation.py

```diff
@@ -2571,6 +2643,11 @@ def _scale_operation(self, op, *args):
         *args
             Any arguments required for the operator (typically, what is to
             be multiplied with, divided by).
+        scaled_base : bool, optional
+            Whether the base was scaled the same way. This affects whether
+            differential components should be scaled. For instance, a differential
+            in longitude should not be scaled if its spherical base is scaled
+            in radius.
         """
         scaled_attrs = [op(getattr(self, c), *args) for c in self.components]
```

Comment:
Elsewhere you use generator comprehension, not list comprehension. I think the newer pattern is better.

```suggestion
        scaled_attrs = (op(getattr(self, c), *args) for c in self.components)
```